lib/ddtrace/contrib/rails/utils.rb

```diff
@@ -50,6 +50,10 @@ def self.adapter_name
           normalize_vendor(connection_adapter)
         end
 
+        def self.database_name
```

Comment:
considering this method is just an alias, is there any reason for this additional level of indirection?

Reply:
I don't think there's much reason beyond just having naming and structure consistent with `adapter_name`. We could reduce this method to an alias or simply remove it altogether.